site-src/overrides/main.html

```diff
@@ -0,0 +1,8 @@
+{% extends "base.html" %}
```

Comment:
This adds a banner on versions that aren't latest, to point users to latest in case they don't intend to be on this other branch.